Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -124,12 +124,17 @@ class BrowserViewController: UIViewController {
 
     override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransitionToSize(size, withTransitionCoordinator: coordinator)
-
-        displayedPopoverController?.dismissViewControllerAnimated(true, completion: nil)
-
+        
         guard let displayedPopoverController = self.displayedPopoverController else {
```

Comment:
how about we move the second `guard` statement into a `where` clause on the first?
```
guard let displayedPopoverController = self.displayedPopoverController where displayedPopoverController.isViewLoaded() && displayedPopoverController.view.window != nil else {
    return self.displayedPopoverController = nil
}
```